### DISCUSSION TITLE
Report file is to large

### DISCUSSION THREAD

**Discussion by dereksnyders** [allure-framework/allure2#2646] [Bug Reports]
Hi

We have 579 tests and some of them have screenshots that needs to be attach to the report now that is all working fine but the problem is that when we Publish the report it gives us this log
" Report file is too large (31.189369201660156MB), uploaded in 4 chunks"

![image](https://github.com/user-attachments/assets/05c8919a-c9c4-445a-b67d-81b435bcac41)

When i open the report after the build the report looks like this
![image](https://github.com/user-attachments/assets/19fbdd3f-30fb-461b-9060-8cfc18c3aaac)

Can you assist me with this issue please?.

**Comment by epszaw** [comment]
Hello, @dereksnyders! Will check the issue as soon as possible

**Comment by ymouhat** [comment]
Hi @epszaw 
is by any chance someone could have a look at it ?
I suspect similar issues occuring on our side due  too many scenario failing an having too many screenshot taken . 
consequently it indicates the report has being published in azure devops pipeline but when you go in the allure tab, there is not report displayed. 
only for those that passed

<img width="1362" height="674" alt="image" src="https://github.com/user-attachments/assets/c7feb8e1-a7b8-4db9-b419-022794646383" />
as you can see some are missing 

<img width="478" height="1098" alt="image" src="https://github.com/user-attachments/assets/11b504e2-dba4-4afc-9d2e-05d8671c817e" />

and those which are missing 

<img width="2244" height="612" alt="image" src="https://github.com/user-attachments/assets/090276f9-e361-4135-9c01-25b32cab3c19" />


We are using PublishAllureReport  using version 1.5.0
with  allureVersion: '2.29.0'

**Comment by BenFison** [comment]
Hi @epszaw,
We also have an issue where reports generate abnormally large:
Error: The report file is too large (2819.804174423218MB), the maximum possible size to assemble the report in browser is 512MB.
We do not attach videos or other large attachments and the number of screenshots taken would not produce 2819MB of data. The same pipeline has been run multiple times without issue.
I have tried updating to the latest version of Allure. 
Any assistance would be appreciated.
Thank you

**Comment by NebelskyiSage** [comment]
@epszaw I also found out this issue, any updated on that please ?
